File: PHP/quadratic-equation.php
<?php

/**
 * @return array An array of two elements containing roots in any order
 */
function findRoots($a, $b, $c)
{
    $firstValue = (-$b + sqrt( pow($b, 2) - (4 * $a * $c))) / (2 * $a);
    $secondValue = (-$b - sqrt( pow($b, 2) - (4 * $a * $c))) / (2 * $a);
    return [$firstValue, $secondValue];
}

print_r(findRoots(2, 10, 8));

File: C#/Quadratic Equation/QuadraticEquation.cs
using System;

public class QuadraticEquation
{
    public static Tuple<double, double> FindRoots(double a, double b, double c)
    {      
        double firstValue = ((-b + Math.Sqrt(Math.Pow(b, 2) - (4 * a * c))) / (2 * a));
        double secondValue = ((-b - Math.Sqrt(Math.Pow(b, 2) - (4 * a * c))) / (2 * a));
      
        return new Tuple<double, double>(firstValue, secondValue);
    }

    public static void Main(string[] args)
    {
        Tuple<double, double> roots = QuadraticEquation.FindRoots(2, 10, 8);
        Console.WriteLine("Roots: " + roots.Item1 + ", " + roots.Item2);
    }
}
